.azure-pipelines/pipelines.yml

```diff
@@ -451,6 +451,19 @@ stages:
         GCS_ARTIFACT_BUCKET: $(GcsArtifactBucket)
       condition: always()
 
+  - job: complete
+    displayName: "linux_x64"
+    dependsOn: ["bazel", "coverage"]
+    pool:
+      vmImage: "ubuntu-20.04"
+    condition: |
+      and(
+        in(dependencies.bazel.result, 'Succeeded', 'SucceededWithIssues', 'Skipped'),
```

Comment:
Can we comment about this job and how it works inline?  I would assume that SucceededWithIssues would be bad and not grouped with the other two.
/wait

Reply:
i think we need `SucceededWithIssues` - not sure exactly what it does but this condition is functionally equivalent to using the `succeeded()` queue time var

that is my interpretation of:

`succeeded()` `==` `in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues')`

here https://learn.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#succeeded

Reply:
comment added